fluentd/wait_for_es_version.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/bash
+
+ruby wait_for_es_version.rb $@
```

Comment:
we need to do this because of how ruby is added to the image. this will be the entry point for the init container

Reply:
I think you can also do this with `#!/usr/bin/env ruby`